lib/cosmos/tools/launcher/launcher.rb

```diff
@@ -102,6 +96,7 @@ def initialize_central_widget
             "font-family:#{config.tool_font_settings[0]}; " \
             "font-size:#{config.tool_font_settings[1]}px"
           label.setStyleSheet(stylesheet)
+          label.setObjectName("ButtonLabel")
```

Comment:
I added some ObjectNames to make it easier to select the various things. I'd probably do the same for all the tools if you think this is a good change.